recordset/recordset.js

```diff
@@ -79,10 +79,12 @@
         $scope.navbarBrandImage = (chaiseConfig['navbarBrandImage'] !== undefined? chaiseConfig.navbarBrandImage : "");
         $scope.navbarBrandText = (chaiseConfig['navbarBrandText'] !== undefined? chaiseConfig.navbarBrandText : "Chaise");
 
+        // TODO: Session.login should already be wrapped in a try catch, maybe don't need to wrap this function with the wrapper (Utils.catchWrapper(function() {...}))
         $scope.login = function() {
```

Comment:
I don't think `login` and `logout` functions will even be needed here. Isn't the navbar responsible for these functionalities?